include/ur.py

```diff
@@ -1810,13 +1810,15 @@ class ur_function_v(IntEnum):
     USM_FREE = 110                                  ## Enumerator for ::urUSMFree
     USM_GET_MEM_ALLOC_INFO = 111                    ## Enumerator for ::urUSMGetMemAllocInfo
     USM_POOL_CREATE = 112                           ## Enumerator for ::urUSMPoolCreate
+    USM_IMPORT = 113                                ## Enumerator for ::urUSMImport
```

Comment:
this should go at the end of enum

Reply:
The enum is generated. I agree that it is odd that it places this value in the middle.

Reply:
In this instance I don't think the gap in ID's is problematic but I agree this should be at the end, I've created #587 to track this.